62_CAD/fragment_shader.hlsl

```diff
@@ -372,6 +372,23 @@ float32_t4 calculateFinalColor<true>(const uint2 fragCoord, const float localAlp
     return col;
 }
 
+// TODO: place this MSDF code elsewhere
+float median(float r, float g, float b) {
+    return max(min(r, g), min(max(r, g), b));
+}
+
+float screenPxRange(float2 screenSpaceSizePixels, float2 distanceFieldSizePixels) {
```

Comment:
Let's not bring this function to the builtin hlsl header, because it's application dependant. Let's be flexible like msdfgen about user providing a screenPxRange. But remember to document good there so there is no confusion about what he screenPxRange is. You can rephrase some of my texts from the long comment

Reply:
Done